src/test/java/net/mcreator/integration/generator/GTModElements.java

```diff
@@ -61,45 +61,47 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 			LOG.info("[" + generatorName + "] Testing mod element type generation " + modElementType.getReadableName()
 					+ " with " + modElementExamples.size() + " variants");
 
-			modElementExamples.forEach(generatableElement -> {

```

Comment:
Is this change needed, this diff seems quite hard to read?

Reply:
I need to generate sample elements to find references within, using main methods from other methods seems bad idea. What should I do in this case?

Reply:
I can't read what you did there because diff is broken on github.

But if you want to make changes to this test for things needed in another test, this is not ok

Reply:
> What should I do in this case?

Make new method for this in your test